website/docs/r/cloudbuild_trigger.html.markdown

```diff
@@ -59,6 +74,10 @@ will be expanded when the build is created:
     or resolved from the specified branch or tag.
   * `$SHORT_SHA`: first 7 characters of `$REVISION_ID` or `$COMMIT_SHA`.
 
+* `filename` - (Optional) Specify the path to a Cloud Build configuration file
+in the Git repo. This is mutually exclusive with `build`. By default this is
```

Comment:
It doesn't seem like there's a default in here, so if you change 'by default' to 'Typically', that will be a bit clearer.

Reply:
Ah, agreed. There's no default on the API, you have to specify a value. It's the web portal that suggests you use cloudbuild.yaml.